File: java/getter.java


public class getter {
    // Atributo privado
    private String nombre;
    
    // Constructor
    public getter(String nombre) {
        this.nombre = nombre;
    }
    
    // Getter para el atributo 'nombre'
    public String getNombre() {
        return nombre;
    }
    public static void main(String[] args) {
        // Crear una instancia de Persona
        getter persona = new getter("Juan");

        // Usar el getter para obtener el valor del atributo 'nombre'
        System.out.println("Nombre: " + persona.getNombre());
        
    }
}

File: clases y trabajos/DART/ejemplos/getter.dart
class Coche {
  // Atributo privado
  final String _modelo;

  // Constructor
  Coche(this._modelo);

  // Getter para el atributo 'modelo'
  String get modelo => _modelo;
}

void main() {
  // Crear una instancia de Coche
  var miCoche = Coche('Toyota Corolla');

  // Usar el getter para obtener el valor del atributo 'modelo'
  print('Modelo del coche: ${miCoche.modelo}');
}